pkg/apiserver/event/event.go

```diff
@@ -57,6 +59,7 @@ func Register(r *gin.RouterGroup, s *Service) {
 	// TODO: add more api handlers
 	endpoint.GET("", s.listEvents)
 	endpoint.GET("/dry", s.listDryEvents)
+	endpoint.GET("/get", s.getEvent)
```

Comment:
I recommend following the restful API, use the format of /events/id to get a single event.

Reply:
ok, i will update it.

Reply:
Due to the `/dry` route will conflict with `/:id` and it's no possible to change the `/dry`(Because of compatibility) for now. We will still use `/get` to get a single event.

Reply:
how about `/events/get/:id` ?

Reply:
or `events/detail/:id`?